kubernetes-csi-external-attacher-4.3.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: kubernetes-csi-external-attacher-4.3
   version: 4.3.0
-  epoch: 19
+  epoch: 20
```

Comment:
We don't need epoch bump if we are only adding tests unless we really want to build the package.

Reply:
It fails in CI without an epoch bump.

Reply:
Curious why that's the case? But given it fails I think a bump is fine here.